raiden/api/python.py

```diff
@@ -736,7 +736,7 @@ def get_raiden_internal_events_with_timestamps(self, limit: int = None, offset:
 
     def get_blockchain_events_network(
             self,
-            registry_address: typing.PaymentNetworkID,
```

Comment:
I think that was correct before, did this cause problems?

Reply:
Thanks for finding I was going to modify 99% of similar things instead of fixing 1%.

Reply:
When I reverted this change, I had to add one type casting instead. This one makes sense to me.  https://github.com/raiden-network/raiden/pull/3144/commits/af11ca0c2d73202178cbdccbd30eaa0b42e816e6#diff-62070ecf9e8fed09dea509b5b3957664R87

Reply:
Yes, I agree. This way we have a special type in the codebase and only convert it at the boundary.